app/_landing_pages/dev-portal/self-service.yaml

```
@@ -0,0 +1,213 @@
+metadata:
+  title: "Developer self-service"
+  content_type: landing_page
+  description: "Enable self-service registration flows for developers and applications using authentication strategies and {{site.konnect_short_name}} application auth."
+  products:
+    - dev-portal
+  tags:
+    - application-registration
+    - authentication
+  breadcrumbs:
+    - /dev-portal/
+
+rows:
+
+  - header:
+      type: h1
+      text: "Developer self-service and app registration"
+      sub_text: Enable self-service registration flows for developers and applications using authentication strategies and {{site.konnect_short_name}} application auth.
+
+  - columns:
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: |
+                    {{site.konnect_short_name}} Dev Portal provides flexible options for controlling access to content and APIs. 
+                    When combined with a [Gateway Service](/gateway/entities/service/), developers visiting a Dev Portal can sign up, create an application, register it with an API, and retrieve API keys without intervention from Dev Portal administrators. 
+
+                    Developer self-service consists of two main components:
+                    * **User authentication:** Allows users to access your Dev Portal by logging in. You can further customize what logged in users can see using RBAC.
+                    * **Application registration:** Allows developers to use your APIs using credentials and create applications for them.
+
+                    To enable developer self-service, do the following:
+                    1. Enable user authentication by navigating to **Settings > Security** in your Dev Portal.
+                       
+                       Developer sign ups and application creation require admin approval by default, which can also be configured in the Dev Portal security settings.
+                       For private Dev Portals, user authentication is enabled by default, and the default application auth strategy is key authentication.
+                    1. Configure an application authentication strategy by navigating to **Settings > Security**.
+                    1. Link an [API to a Gateway Service {% new_in 3.6 %}](/dev-portal/apis/#gateway-service-link).
```

```suggestion
                    1. Link an [API to a Gateway Service](/dev-portal/apis/#gateway-service-link).
```
The section at https://deploy-preview-1940--kongdeveloper.netlify.app/dev-portal/apis/#gateway-service-link doesn't have a new_in badge. 

I'm actually not sure that "new_in" is applicable here. As a Konnect/Dev Portal user, I don't know that I would know about data plane versioning (it might not even be the same person administering both things). I'd recommend removing the new_in badge on this landing page, then adding a line in the linked doc that says something like "available for Data Planes running Gateway version 3.6+"